metaci/repository/models.py

```diff
@@ -42,7 +42,7 @@ def get_absolute_url(self):
         return reverse('branch_detail', kwargs={'owner': self.repo.owner, 'name': self.repo.name, 'branch': self.name})
 
     def __unicode__(self):
-        return u'[{}] {}'.format(self.repo.name, self.name)
+        return u'{}'.format(self.name)
```

Comment:
This was good to have for selecting a branch in a dropdown menu, e.g. when running a plan. Can we change this back?

Reply:
Just saw this isn't used in plan run view anyway, so it seems safe to change.